curriculum/challenges/english/08-data-analysis-with-python/numpy/loading-data-and-advanced-indexing.md

```diff
@@ -11,16 +11,16 @@ videoId: tUdBZ7pF8Jg
 
 Given a file named `data.txt` with these contents:
 
-<pre>
+```
```

Comment:
All code blocks must specify a language.  This is not code, so what is wrong with `pre` tags here?

Reply:
It seems I lost one, adding `pre` there :+1: